spec/myst/time_spec.mt

```diff
@@ -74,8 +130,75 @@ describe("Time#-") do
     assert(t1 - t2 == -1.0)
   end
 
+  it("gives fractional seconds") do
+    t1 = %Time{2017, 1, 2, 3, 4, 5, 7}
+    t2 = %Time{2017, 1, 2, 3, 4, 6, 8}
+    assert(t2 - t1 == 1.000000001)
+  end
+
   it("raises if a non Time type is passed") do
     t1 = %Time{2017, 1, 2, 3, 4, 5}
     expect_raises { t1 - nil }
   end
 end
+
+describe("Time#year") do
+  it("returns the year") do
+    t = %Time{2017, 1, 2, 3, 4, 5}
+    assert(t.year == 2017)
+  end
+end
+
+describe("Time#month") do
+  it("returns the year") do
```

Comment:
probably a copy/paste thing: this says "returns the year" instead of "returns the month". Same in the following specs.